packages/environment-check/src/types.ts

```diff
@@ -22,6 +36,20 @@ export interface Environment {
     basDevSpace?: string;
 }
 
+export interface ToolsExtensions {
```

Comment:
Interface `ToolsExtensions` looks good, but I would have an optional property in `Environment` with this interface. Logically extension and module versions belong to the environment. Right now we read and fill this information when running in VSCode only, but at some point we might collect the same information in BAS.

Reply:
Updated